.github/workflows/ci.yml

```diff
@@ -29,10 +29,34 @@ jobs:
           apk add --no-progress --update git
           mix deps.get
       - run: mix test
+      # - run: MIX_ENV=docs mix inch.report
+
+  test-installer:
+    runs-on: ubuntu-latest
+    container:
+      image: hexpm/elixir:${{ matrix.elixir }}-erlang-${{ matrix.otp }}-alpine-3.12.0
+      env:
+        ELIXIR_ASSERT_TIMEOUT: 2000
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - elixir: 1.11.0
+            otp: 21.3.8.18
+          - elixir: 1.11.0
+            otp: 23.1.1
+    steps:
+      - uses: actions/checkout@v2.3.1
+      - name: Install Dependencies
+        run: |
+          mix local.rebar --force
+          mix local.hex --force
+          apk add --no-progress --update git
+          mix deps.get
```

Comment:
Instead of having this as a separate group, we can move the installer test with the integration tests, because they always need to run in the same Elixir versions. :)